docs/examples/005-viz-with-holoviz.ipynb

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
<p>Add a small text or comment to the beginning like "Using <code>matplotlib</code> as the backend"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>